cylc/flow/network/resolvers.py

```diff
@@ -655,7 +617,16 @@ async def _mutation_mapper(
             self.schd.get_command_method(command)
         except AttributeError:
             raise ValueError(f"Command '{command}' not found")
-        self.schd.command_queue.put((command, tuple(kwargs.values()), {}))
+        if command != "put_messages":
+            log_msg = f"[command] {command} "
+            user = meta.get('auth_user', self.schd.owner)
+            if user != self.schd.owner:
+                log_msg += (f"(issued by {user})")
```

Comment:
Minor point to avoid logging trailing whitespace:

```suggestion
            log_msg = f"[command] {command}"
            user = meta.get('auth_user', self.schd.owner)
            if user != self.schd.owner:
                log_msg += (f" (issued by {user})")
```